Kajabi/sage-archived
# Checkbox styling

## Description
Adds markup and styling for checkboxes

### Screenshots
<img width="396" alt="Screen Shot 2020-02-24 at 8 29 04 AM" src="https://user-images.githubusercontent.com/1175111/75170952-c15cc680-56df-11ea-84e9-cbb7af04b727.png">

## Test notes
- This initial styling does not include the error state.

### Steps for testing
- Start Sage with `rails s`
- Navigate to "Elements" in the navigation
- Verify that checkboxes match the [Figma design spec](https://www.figma.com/file/xlUFoRTjsFePavVT85SmLL/Sage-Design-System-v1.0?node-id=634%3A4745&viewport=293%2C-2215%2C2.684436321258545)


## Related issues
- Closes #40
